apps/css-workshop/pages/index.html

```diff
@@ -28,7 +28,7 @@
     >
     <meta
       property="og:image"
-      content="https://raw.githubusercontent.com/iTwin/iTwinUI/gh-pages/backstop/assets/logo.png"
+      content="./assets/logo.png"
     >
```

Comment:
Not sure if the `og:image` property is working. Because when I pasted the [deploy preview URL](https://itwin.github.io/[iTwinUI](https://itwin.github.io/iTwinUI/1726/css/)/1726/css/) in Teams, the link tile didn't include the iTwinUI logo.

<img width="502" alt="image" src="https://github.com/iTwin/iTwinUI/assets/45748283/1bd31fba-1dbf-45f8-9d95-9278de7e5c5c">

But it does include the logo for other websites with `og:image`. E.g. [MUI](https://mui.com/material-ui/getting-started/):

<img width="500" alt="image" src="https://github.com/iTwin/iTwinUI/assets/45748283/cba62e3a-0c24-4721-a333-05e700acb48d">

Reply:
Turns out `og:image` requires an absolute URL.  Updated although it likely won't work until the image is deployed to `main`.

Reply:
When inspecting the network tab, I noticed that the favicon was downloaded with this url: `https://itwin.github.io/iTwinUI/css/assets/favicon-3ea24d8f.ico`.

<img width="530" alt="image" src="https://github.com/iTwin/iTwinUI/assets/45748283/b7048724-33bb-4511-ba21-4884dbdc635a">

Does vite automatically handle adding that hash suffix to the url?

Reply:
🤷🏻‍♂️